index.html

```diff
@@ -39,7 +39,7 @@ <h3 class="text-muted">Chromebrew</h3>
       </div>
 
       <div class="row marketing">
-        <div class="col-lg-6">
+        <div class="col-6">
```

Comment:
@jam7 `col-6` is a valid option. See the [grid system manual](https://v4-alpha.getbootstrap.com/layout/grid/#grid-options).